.azure-pipelines/unix-build.yml

```diff
@@ -7,9 +7,8 @@ steps:
       if [[ $(enable_xtl_complex) == 1 ]]; then
         conda install xtl -c conda-forge
       fi
-      if [[ $(avx512) == 1 ]]; then
-        conda install gcc_linux-64 gxx_linux-64;
-        sh $(Build.SourcesDirectory)/install_sde.sh;
+      if test "x$(avx512)" = "x1" ; then
```

Comment:
What is the benefit of this test compared to the previous one?

Reply:
That's an old idiom, it's a reflex: https://stackoverflow.com/questions/6852612/test-for-empty-string-with-x

Reply:
Ok, thanks for the explanation!